File: 1051-height-checker/1051-height-checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        heightsCopy = heights.copy()
        heights.sort()
        outOfOrder = 0
        for i in range(len(heights)):
            if heights[i] != heightsCopy[i]:
                outOfOrder += 1
        return outOfOrder
        

File: 1051-height-checker/1051-height-checker.js
/**
 * @param {number[]} heights
 * @return {number}
 */
var heightChecker = function(heights) {
    const heightsCopy = [...heights]
    heights.sort((a,b) => a-b)
    let result = 0
    for(let i = 0; i < heights.length; i++){
        if(heights[i] != heightsCopy[i]){
            result += 1
        }
    }
    return result
};